lib/aff4.py

```diff
@@ -2393,6 +2405,10 @@ def Initialize(self):
       self.size = 0
       self.content_last = None
 
+    # This is the configurable default length for allowing Read to be called
+    # without a specific length
+    self.max_unbound_read = config_lib.CONFIG["Server.max_unbound_read_size"]
```

Comment:
Oh duh, sorry, I totally looked at the code wrong.  I am working from wandering around at RSA, I will get this fixed soon. :)